### PR TITLE
add New Relic Dashboard plugin in marketplace

### PR DESCRIPTION
**Pull request by mufaddal7**
Signed-off-by: mufaddal motiwala <mufaddalmm.52@gmail.com>

Add New Relic Dashboard plugin in the marketplace 

image

### CODE REVIEW

File: microsite/data/plugins/newrelic-dashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: New Relic Dashboard
+author: '@mufaddal7'
+authorUrl: https://github.com/mufaddal7
+category: Monitoring
+description: Easily view your New Relic Dashboards in backstage , creates real-time snapshot of your dashboards
```
Comment (Rugvip) [inline microsite/data/plugins/newrelic-dashboard.yaml:6]:
```suggestion
description: Easily view your New Relic Dashboards in Backstage, via real-time snapshots of your dashboards
```

Rewording is optional, but should get rid of the extra space :grin:

**Review by Rugvip — COMMENTED** [review]
Thanks! just a minor nit

File: microsite/data/plugins/newrelic-dashboard.yaml (hunk 1 above)
Comment (mufaddal7) [reply microsite/data/plugins/newrelic-dashboard.yaml]: Done 👍
Thanks for the suggestion

**Review by Rugvip — APPROVED** [review]
:shipit: 🎉

**Pull request merged**